template.md

```diff
@@ -10,6 +10,20 @@
 - **Location**: CITY, COUNTRY
 - **Attendee information**: LINK TO REFLECTOR
 
+```mermaid
+---
+displayMode: compact
+---
+gantt
+    title: TBD Meeting
+    dateFormat  YYYY MM DD HH mm Z
```

Comment:
```suggestion
```

this is supposed to represent the date format you are inputting, and this doesn't match.  might as well just use the default iso-8601